hack/templates/values/values.template

```diff
@@ -309,14 +393,20 @@ sriovDevicePlugin:
   #     limits:
   #       cpu: "150m"
   #       memory: "100Mi"
+  # -- Optional `resource requests and limits <https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/>`_
```

Comment:
the resources below actually defines the device plugin configuration. its not requests and limits for the sriovdp-container.